### DISCUSSION TITLE
Custom logo colour?

### DISCUSSION THREAD

**Discussion by louisrousseau** [mainsail-crew/mainsail#1853] [Help]
Is there a way to recolor `sidebar-logo.svg` to follow the logo color the way the stock Mainsail one does? By using CSS perhaps?

**Comment by meteyou** [comment]
Right now, there is no option for it.

**Comment by meteyou** [comment]
hey @louisrousseau! i added this feature to Mainsail and released it now: https://github.com/mainsail-crew/mainsail/pull/1856

**Reply by louisrousseau** [reply]
Managed to hack one of the Voron theme sidebar logo SVGs with `fill="var(--color-logo, #ec3324)"` and it works, thanks!
